asApp/Db/save_restoreStatus.db

```diff
@@ -80,8 +80,22 @@ record(stringout, "$(P)SR_0_Name") {
   field(DTYP, "Soft Channel")
 }
 
-record(longout, "$(P)SR_0_State") {
+record(mbbo, "$(P)SR_0_State") {
+  field(DESC, "Save method for pass 0")
   field(DTYP, "Soft Channel")
+  field(NOBT, "5")
+  field(ZRVL, "0")
+  field(ONVL, "1")
+  field(TWVL, "2")
+  field(THVL, "4")
+  field(FRVL, "12")
+  field(FVVL, "16")
+  field(ZRST, "None")
+  field(ONST, "Periodic")
+  field(TWST, "Triggered")
+  field(THST, "Timer")
+  field(FRST, "Monitored")
+  field(FVST, "Manual")
```

Comment:
Unless I misunderstand something, this does not do what you are wanting. What you are trying to do is give meaningful enumerative names to the various values for autosave's methods. However, those should map to the actual values, not use the `ZRVL`,  `ONVL`, etc.

See specifically that the autosave code that writes these values (e.g. https://github.com/epics-modules/autosave/blob/d7bcdf2ae4163cd554ea139c902a88dbe558e065/asApp/src/save_restore.c#L1117) writes via CA the value (e.g. 12), but if we write via CA the value 12 we do not get "Monitored", but an empty string.

Reply:
The issue was with me using `mbbo`.

From our discussion, I now instead set this up as new (`mbbi`) PVs.

```
(e3) anderslindh@ics-alo-vm-ubuntu:~$ pvput alo:SR_0_State 12
Old : <undefined>              0 INVALID DRIVER UDF
New : 2025-11-24 14:58:45.044  12
(e3) anderslindh@ics-alo-vm-ubuntu:~$ pvget alo:SR_0_StateEnum
alo:SR_0_StateEnum 2025-11-24 14:58:45.044  (4) Monitored
```